docs/docs/concepts/foundation/communication/public_private_calls/slow_updates_tree.md

```diff
@@ -0,0 +1,95 @@
+---
+title: Privately access Public State
+---
+
+In Aztec, private and public execution environments are completely separate and operate with distinct state management. In the previous section we learned that functions can call private functions, and public functions can save messages to a merkle tree to then be read by a private function.
+
+It is also possible to access persistent data across both private and public state using a **slow updates tree**, which we will talk about on this page. Please note that we are still experimenting with this feature and it is not something that has ever been achieved before.
```

Comment:
Uhh so slow updates doesn't let you read public state due to race conditions. And we **need** to be very clear to prevent people from thinking this:

```suggestion
But, what about historical public data? Through a **slow updates tree**, you can have private functions access historical public state. Please note that we are still experimenting with this feature.
```